2020-01-Winter/0_arrays_sorting/Merge_Sorted_Arrays/mergeSortArr.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <vector>
+
+std::vector<int>& merge(std::vector<int>& nums1, int m, std::vector<int>& nums2, int n)
+{
+  int backOfArray = nums1.size()-1;
+
+  while(m-1 > 0)
+  {
+    if(nums1[m-1] < nums2[n-1])
+    {
+      nums1[backOfArray] = nums2[n-1];
+      std::cout << n;
```

Comment:
Don't need this, and the lack of delimiter makes the test output look confusing.
```suggestion
```